R/sp.R

```diff
@@ -330,7 +330,10 @@ CRS_from_crs = function(from) {
 			sp::CRS(NA_character_)
 		else
 			sp::CRS(from$proj4string)
-	if (!is.null(from$wkt) && !is.na(from$wkt))
-		comment(ret) = from$wkt
+        if (CPL_proj_version() < "6.0.0" || 
+                    CPL_gdal_version() < "3.0.0") {
+	    if (!is.null(from$wkt) && !is.na(from$wkt))
+		comment(ret) = sp::CRS(SRS_string=from$wkt)
```

Comment:
It seems you assign an object of class `CRS` to a comment; I always thought comments had to be text.

Reply:
Added comment. There was a logic error in the condition too. I've committed & pushed, as well as expanding the crs/CRS coercion section in tests/crs.R.